examples/ola-recorder.cpp

```diff
@@ -200,9 +174,24 @@ int VerifyShow(const string &filename) {
  * Playback a recorded show
  */
 int PlaybackShow() {
-  ShowPlayer player(FLAGS_playback.str());
+  const string filename = FLAGS_playback.str();
+  if (!FLAGS_no_verify) {
+    // Verify the show and print a summary before running
+    string summary;
+    const int verified = VerifyShow(filename, &summary);
+    // Printing a newline first makes this output look better in interactive
+    // terminal logs.
+    OLA_INFO << endl << summary;
+    if (verified != ola::EXIT_OK) {
+      // Show did not pass verification
+      return verified;
```

Comment:
We should print an error here, or do we get one from verify?

Reply:
Verify handles printing errors (it's the same method used when running with `--verify`); I'll add a comment about that.